torchft/process_group.py

```diff
@@ -602,6 +628,10 @@ def configure(self, store_addr: str, rank: int, world_size: int) -> None:
         )
         self._p.start()
 
+        # fetch the status of the PG init
+        # if an exception was returned _get will throw
+        assert _get(rx, self._timeout) is None
+
```

Comment:
If `_get` will throw the Exception, why do we need this assertion?

Reply:
just to make sure we consume the value -- doesn't really matter what the output is as long as it's consistent